docs/source/upcoming_release_notes/1352-hide-pa1k4-targets.rst

```diff
@@ -0,0 +1,30 @@
+1352 hide-pa1k4-targets
+#################
+
+API Breaks
+----------
+- N/A
+
+Library Features
+----------------
+- N/A
+
+Device Features
+---------------
+-  for "PA1K4"('CalibrationAxis'),change target count from 7 to 5 to  hide 6 and 7
```

Comment:
```suggestion
- For ``PA1K4`` (`CalibrationAxis`), change target count from 7 to 5 to hide targets 6 and 7
```

Reply:
I'm just going to force this through the way I suggested because I have pain later when making release notes if the notes aren't right